PROJECT_PROGRESSION.md

```diff
@@ -116,6 +116,7 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
 - [ ] Add Project to the [CPC repo's README.md](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/README.md#incubating-projects) as an Incubating project
 - [ ] List trademarks the project commits to transfer to the OpenJS Foundation following graduation
```

Comment:
Maybe this line could then be rephrased to make it more implicit that any pending trademark should be transferred from the original project?